collective/embedly/setuphandlers.py

```diff
@@ -16,8 +16,8 @@
 })
 
 
+#implementer(INonInstallable)
```

Comment:
It seems this comment is a mistake :)
Should it be:
```suggestion
@implementer(INonInstallable)
```
?

Reply:
OMG! yes! Thanks @ale-rt !